From 604d24c8a7370e10eadf336a3f867a7f9174a145 Mon Sep 17 00:00:00 2001
From: Zidan Wang <zidan.wang@freescale.com>
Date: Fri, 27 Nov 2015 11:15:38 +0800
Subject: [PATCH 0702/5242] MLK-11915-06 ASoC: fsl_esai: fix missing break in
 switch

commit  41db9ed4f58ffb93f1244dd4d4f224089e092ba8 from
https://source.codeaurora.org/external/imx/linux-imx.git

fix missing break in switch. Reported by Coverity.

Signed-off-by: Zidan Wang <zidan.wang@freescale.com>
(cherry picked from commit ec9d74d9258e3320ec68a1f977932cb20ace4ca2)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 sound/soc/fsl/fsl_esai.c |    1 +
 1 file changed, 1 insertion(+)

diff --git a/sound/soc/fsl/fsl_esai.c b/sound/soc/fsl/fsl_esai.c
index 58df003..b28ee9c 100644
--- a/sound/soc/fsl/fsl_esai.c
+++ b/sound/soc/fsl/fsl_esai.c
@@ -252,6 +252,7 @@ static int fsl_esai_set_dai_sysclk(struct snd_soc_dai *dai, int clk_id,
 		break;
 	case ESAI_HCKT_EXTAL:
 		ecr |= ESAI_ECR_ETI;
+		break;
 	case ESAI_HCKR_EXTAL:
 		ecr |= ESAI_ECR_ERI;
 		break;
-- 
1.7.9.5

